components/error-summary/src/fixtures.js

```diff
@@ -8,7 +8,8 @@ import TextArea from '@govuk-react/text-area';
 import ErrorSummary from '.';
 
 export const heading = 'Message to alert the user to a problem goes here';
-export const description = 'Optional description of the errors and how to correct them';
+export const description =
```

Comment:
this change is because of the prettier default line width of 80 characters

i think i still prefer 120

Reply:
ok will set prettier to use 120